vnext/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -167,7 +169,7 @@ class ViewShadowNode : public ShadowNodeBase
   bool m_onClick = false;
   int32_t m_tabIndex = std::numeric_limits<std::int32_t>::max();
   AccessibilityRoles m_accessibilityRole = AccessibilityRoles::None;
-  bool m_accessibilityStates[AccessibilityStates::CountStates] = { false, false };
+  bool m_accessibilityStates[AccessibilityStates::CountStates] = { 0 };
```

Comment:
>0 [](start = 67, length = 1)

maybe you can remove 0. or instead of '0', it should be false

Reply:
Fixed.